ChemCheck/upload/views.py

```diff
@@ -93,7 +98,7 @@ def ace(request, pk, filetype):
     f = f.path
 
     filename = os.path.split(f)[-1]
-    content = open(f, "r").read()
+    content = strip_nonascii(open(f, 'r', errors='ignore').read())
```

Comment:
This looks like a syntax error, plus the same note from before about using `with`.